SpotifyAPI/Local/Models/Track.cs

```diff
@@ -102,32 +109,31 @@ public string GetAlbumArtUrl(AlbumArtSize size)
         /// Returns a Bitmap of the album cover in the provided size asynchronous
         /// </summary>
         /// <param name="size">AlbumArtSize (160,320,640)</param>
+        /// <param name="proxyConfig">Optional proxy settings</param>
         /// <returns>A Bitmap, which is the albumart</returns>
-        public async Task<Bitmap> GetAlbumArtAsync(AlbumArtSize size)
+        public async Task<Bitmap> GetAlbumArtAsync(AlbumArtSize size, ProxyConfig proxyConfig = null)
         {
-            using (WebClient wc = new WebClient())
+            var data = await GetAlbumArtAsByteArrayAsync(size, proxyConfig).ConfigureAwait(false);
+            using (MemoryStream ms = new MemoryStream(data))
```

Comment:
Don't we need another `null` check here? data could be null, IDK how memory stream handles it tho